account_banking_pain_base/README.rst

```diff
@@ -56,13 +56,14 @@ Contributors
 
 * Alexis de Lattre
 * Pedro M. Baeza
-* Stéphane Bidoul		<stephane.bidoul@acsone.eu>
+* Stéphane Bidoul              <stephane.bidoul@acsone.eu>
```

Comment:
Curious why this space

Reply:
My editor replaces tabulators for spaces.

Reply:
Well, we can remove that spaces if you want, but no special problem.